nx.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Does this mean that we are using `nx` now?

Reply:
nx is part of lerna 6 now. So, yea we are using nx.